Mathlib/Tactic/Abel.lean

```diff
@@ -495,21 +509,7 @@ def abelNFLocalDecl (s : IO.Ref AtomM.State) (cfg : AbelNF.Config) (fvarId : FVa
   | none => replaceMainGoal []
   | some (_, newGoal) => replaceMainGoal [newGoal]
 
-/-- Unsupported legacy syntax from mathlib3, which allowed passing additional terms to `abel`. -/
-syntax (name := abel_term) "abel" (&" raw" <|> &" term")? (location)? : tactic
-/-- Unsupported legacy syntax from mathlib3, which allowed passing additional terms to `abel!`. -/
-syntax (name := abel!_term) "abel!" (&" raw" <|> &" term")? (location)? : tactic
```

Comment:
Is this still useful to keep around? It caused troubles with the `macro_rules` for `abel` below

Reply:
This line effectively functions as a TODO to remind us that the current version of `abel` is not feature-complete relative to the lean 3 implementation. If we want to remove it, we should either create an issue to keep tracking this, or officially decide that we don't want this feature anymore.

Mathport doesn't work on mathlib anymore (we decided to drop support for mathport when the `#align` lines were deleted), so the original purpose of these lines is gone and we can safely transfer everything to github issues.

Reply:
Thank you for the explanation. I've added a comment to the module docstring to capture this:


> ## Future work
> 
> * In mathlib 3, `abel` accepted addtional optional arguments:
>   ```
>   syntax "abel" (&" raw" <|> &" term")? (location)? : tactic
>   ```
>   It is undecided whether these features should be restored eventually.